src/slider.scss

```diff
@@ -0,0 +1,144 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-slider;
+$handle-desktop-dimensions: 1.25rem;
+$handle-hit-area-desktop-dimensions: 2rem;
+$handle-mobile-dimensions: 1.625rem;
+$handle-hit-area-mobile-dimensions: 2.75rem;
+
+.#{$block} {
+  @include fd-reset();
+
+  @include fd-disabled() {
+    cursor: not-allowed;
+  }
+
+  min-width: 4rem;
+  max-width: 100%;
+  padding: 1rem 0;
+  position: relative;
+
+  &:focus {
+    outline: none;
+  }
+
+  &__wrapper {
+    @include fd-reset();
+
+    position: relative;
+  }
+
+  &__track {
+    @include fd-reset();
+
+    height: var(--fdSlider_Track_Height);
+    width: 100%;
+    background-color: var(--sapField_BorderColor);
+    border-radius: var(--fdSlider_Track_BorderRadius);
+    position: relative;
+    z-index: 1;
+  }
+
+  &__track-range {
+    @include fd-reset();
+
+    position: relative;
+    height: 100%;
+    border-radius: 0.25rem;
+    background-color: var(--fdSlider_Track_Active_Background);
+    border: 0.0625rem solid var(--fdSlider_Track_Active_Border);
+  }
+
+  &__tick-wrapper {
+    @include fd-reset();
+
+    position: absolute;
+    width: 100%;
+    top: -0.4rem;
+  }
+
+  &__ticks {
+    @include fd-reset();
+
+    @include fd-flex() {
+      justify-content: space-between;
+    }
+  }
+
+  &__labels {
+    @include fd-reset();
+
+    @include fd-flex() {
+      justify-content: space-between;
+    }
+
+    position: relative;
+    height: 1rem;
+    top: 0.5rem;
+  }
+
+  &__tick {
+    @include fd-reset();
+
+    position: relative;
+    width: 0.0625rem;
+    height: var(--fdSlider_Tick_Height);
+    background-color: var(--sapField_BorderColor);
+
+    &--in-range {
+      background-color: var(--fdSlider_Track_Active_Border);
+    }
+  }
+
+  &__label {
+    @include fd-reset();
+
+    position: relative;
+    font-size: var(--sapFontSmallSize);
+    color: var(--sapContent_LabelColor);
+  }
```

Comment:
Can we center labels doing this? 
![image](https://user-images.githubusercontent.com/3263744/95878151-9661d880-0d7d-11eb-883f-a8b0df6aca52.png)

Reply:
awesome solution, thank you!